registry/storage/driver/testsuites/testsuites.go

```diff
@@ -619,6 +619,39 @@ func (suite *DriverSuite) TestMoveInvalid(c *check.C) {
 	c.Assert(err, check.NotNil) // non-nil error
 }
 
+// TestMoveLargeStreams tests that a larger file is moved to the storage
+// driver safely.
+func (suite *DriverSuite) TestMoveLargeStreams(c *check.C) {
```

Comment:
This test is nearly identical to `TestMove`. Make a single function with the content size parameter so we don't have to maintain both.